### PR TITLE
Added information about add-ons being uninstalled when the delete cluster command is run.

### PR DESCRIPTION
**Pull request by jonquilwilliams**
Also fixed the `delete cluster` command example and some formatting issues.

JIRA: https://issues.redhat.com/browse/OSDOCS-1763

No preview is possible with this repo. Please review the changes through the PR itself.

@yufchang  @vikram-redhat  @jeremyeder  @okashi18

### CODE REVIEW

File: modules/rosa-deleting-cluster.adoc [hunk 1]
```diff
@@ -1,25 +1,27 @@
...
 
 [id="rosa-deleting-cluster_{context}"]
-= Deleting your cluster
+= Deleting a cluster
 
-You can delete an {product-title} cluster using `rosa`.
+You can delete an {product-title} cluster using the `rosa` CLI.
 
-.Procedure
+If add-ons are installed, the deletion takes longer because the add-ons are uninstalled before the cluster is deleted. The amount of time depends on the number and size of the add-ons.
```
Comment (jeremyeder) [inline modules/rosa-deleting-cluster.adoc]: s/because the/because

File: modules/rosa-deleting-cluster.adoc [hunk 2]
```diff
@@ -1,25 +1,27 @@
...
++
 [source, terminal]
 ----
-$ rosa delete cluster <my-cluster> --watch
+$ rosa delete cluster --cluster=<cluster_name> --watch
 ----
 
-To clean up your CloudFormation stack, enter the following command. Your CloudFormation stack is initially created when you run the `rosa init` command as part of the prerequisites.
-
+. To clean up your CloudFormation stack, enter the following command. Your CloudFormation stack is initially created when you run the `rosa init` command as part of the prerequisites.
```
Comment (jeremyeder) [inline modules/rosa-deleting-cluster.adoc]: I would reverse the order of these sentences.

Comment (bergerhoffer) [inline modules/rosa-deleting-cluster.adoc]: Is the second sentence of "Your CloudFormation stack..." really necessary here? I'm not sure if I see how it is important information when you're deleting it, to know how it was initially created.

If it is important to keep, then I'd consider moving it to after the command, and also using past tense. Maybe something like "Your CloudFormation stack was initially created by using the `rosa init` command as part of the prerequisites."

**Review by bergerhoffer — COMMENTED** [review]
Just a thought on the second step.

File: modules/rosa-deleting-cluster.adoc (hunk 1 above)
Comment (jonquilwilliams) [reply modules/rosa-deleting-cluster.adoc]: Done.

File: modules/rosa-deleting-cluster.adoc (hunk 2 above)
Comment (jonquilwilliams) [reply modules/rosa-deleting-cluster.adoc]: Followed Andrea's suggestion instead.

Comment (jonquilwilliams) [reply modules/rosa-deleting-cluster.adoc]: Deleted the second sentence. Thanks for catching that.

**Comment by bergerhoffer** [comment]
LGTM, merging.

**Pull request merged**